packages/mdl-drawer/README.md

```diff
@@ -0,0 +1,284 @@
+# MDL Drawer
+
+The MDL Drawer component is a spec-aligned drawer component adhering to the
+[Material Design navigation drawer pattern](https://material.google.com/patterns/navigation-drawer.html).
+It implements permanent and temporary drawers. Permanent drawers are CSS-only and require no JavaScript, whereas
+temporary drawers require JavaScript to function, in order to respond to user interaction.
+
+## Installation
+
+> Note: Installation via the npm registry will be available after alpha.
+
+
+## Permanent drawer usage
+
+A permanent drawer is always open, sitting to the side of the content. It is appropriate for any display size larger
+than mobile.
+
+> TODO(sgomes): Give advice on how to hide permanent drawer in mobile.
+
+```css
+body {
```

Comment:
Flex isn't required for the drawer, no, or even for the rest of the content. I'll keep it out of the examples and have the user figure out CSS on their own. They can look at the demos and future "holistic" templates for more guidance.